scripts/check_env_vars.js

```diff
@@ -0,0 +1,131 @@
+const fs = require('fs')
```

Comment:
I think this whole stuff will be automated at some point, so no app will have to think about it. @varl is working on sth right now, I think it will include setting this stuff up for apps as well (similar to how react-scripts do stuff we don't have to worry about), please correct me if I'm wrong @varl

Reply:
Yeah, that's right. Let us revisit this later, I think it's good enough for now. I agree that it's not something we want to duplicate in all apps.